ios/Classes/common/SubmitData.swift

```diff
@@ -1,18 +1,18 @@
 import Foundation
 
 internal struct SubmitData {
-    let paymentData: [String: Any]
-    let extraData: [String: Any?]?
+    let data: [String: Any]
+    let extra: [String: Any?]?
 
```

Comment:
This is auto-generated, right?

Reply:
No, this class was added manually. We can adjust it.

Reply:
I was wondering if we could use Encodable here, but it doesn't seem like we can...